lib/addons/core-addons/index.ts

```diff
@@ -0,0 +1,30 @@
+import { CfnAddon } from "@aws-cdk/aws-eks";
+import { ClusterAddOn } from "../..";
+import { ClusterInfo } from "../../spi";
+
+export class CoreAddOnProps {
+
+    readonly addOnName: string;
+    readonly version: string;
+}
+
+/**
+ * Implementation of EKS Managed add-ons.
+ */
+export class CoreAddOns implements ClusterAddOn {
```

Comment:
It should be singular: `CoreAddOn`. We don't use plurals anywhere and base class is no exclusion. E.g. a generic addon will look like `new CoreAddOn('fluxcd')`.

Reply:
HI @shapirov103 , Done. I have pushed the changes.